pages.id/windows/choco-install.md

```diff
@@ -0,0 +1,36 @@
+# choco install
+
+> Instal satu paket atau lebih dengan Chocolatey.
+> Informasi lebih lanjut: <https://docs.chocolatey.org/en-us/choco/commands/install>.
+
+- Instal satu paket atau lebih paket yang dipisahkan oleh spasi:
+
+`choco install {{nama_paket}}`
```

Comment:
Google translate suggested to translate as `package(s)` as `paket`. Although I request changes please notify me if Google translate translation is incorrect.

Reply:
Yes, This PR must be updated we follow the same in [common-arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)